lib/src/forms/multi_step_form.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:automated_testing_framework/automated_testing_framework.dart';
+import 'package:automated_testing_framework_plugin_flow_control/automated_testing_framework_plugin_flow_control.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:static_translations/static_translations.dart';
+
+class MultiStepForm extends TestStepForm {
+  @override
+  Widget buildForm(
+    BuildContext context,
+    Map<String, dynamic> values, {
+    bool minify = false,
+  }) {
+    /// If this line is removed, [values] will still have a reference to the
+    /// original [steps] List, so modifying that here will modify it in the
+    /// [TestStep] without confirmation.
+    values['steps'] = List.from(values['steps'] ?? []);
+    return Column(
+      children: [
+        if (minify != true)
+          buildHelpSection(
+            context,
+            TestFlowControlTranslations.atf_flow_help_multi_step,
+            minify: minify,
+          ),
+        buildValuesSection(
+          context,
+          [
+            buildEditText(
```

Comment:
What value is this field providing?  It doesn't look to be used other than logs.

Reply:
It's not used as part of the tests but it helps to identify a specific `multi_step`

Reply:
Hmmm...  I'm not a big fan of "user only values", but I'm not totally opposed to them either.  If we're going to introduce those as a concept, about following the naming conventions that Flutter itself uses and call it "debugLabel" instead of "name"?

Also, in this case, it should be excluded on "minify == true" as that's asking the form to only show the required / important parts.